src/cmd/serve.rs

```diff
@@ -597,6 +662,14 @@ pub fn serve(
     println!("Press Ctrl+C to stop\n");
     // Clean the output folder on ctrl+C
     ctrlc::set_handler(move || {
+        println!("\nShutting down...");
+
+        // Signal the server to stop accepting new connections
+        shutdown_flag.store(true, Ordering::Relaxed);
+
+        // Give in-flight requests time to complete (up to 2 seconds)
+        thread::sleep(Duration::from_secs(2));
```

Comment:
that doesn't seem worth it?

Reply:
I can remove this if you don't want it. This won't result in the errors on the console at least.